.github/workflows/test.yml

```diff
@@ -79,6 +79,22 @@ jobs:
       run: |
         pip install git+https://github.com/NeurodataWithoutBorders/pynwb
 
+    - name: Install git-annex (ubuntu)
+      if: startsWith(matrix.os, 'ubuntu')
+      run: |
+        sudo apt-get install git-annex
```

Comment:
The version of git-annex installed this way is out of date.  Change this to:

```
pip install datalad-installer
datalad-installer --sudo ok neurodebian git-annex -m neurodebian
```

Reply:
https://github.com/dandi/dandi-cli/pull/841/commits/90978c7067e1c9c0d4f22789c117e5c1f11c8d64